cli/src/main/kotlin/com/malinskiy/marathon/cli/args/FileConfiguration.kt

```diff
@@ -38,6 +38,7 @@ data class FileConfiguration(
     var includeSerialRegexes: Collection<Regex>?,
     var excludeSerialRegexes: Collection<Regex>?,
     var ignoreFailureRegexes: Collection<Regex>?,
+    var strictFailureRegexes: Collection<Regex>?,
```

Comment:
I know all the other fields don't have any comments, but I think we should start adding them.
The number of various parameters is getting bigger and bigger.
It's worth mentioning `This has higher priority than uncompletedRetriesQuota or amount of strict runs`.

Anothing thing, the current naming uses the word `strict`. The closest analogy is strict run, which means "run until it's green or failed X times". Although the current field has the opposite logic "do not rerun if tests failure contains one of the issues, an event it's under the strict run".

Current me if I am wrong, please.

Therefore I would suggest eliminating a potential misunderstanding.
Maybe `failfastFailureRegexes`, `rerunlessFailureRegexes`or something that reflects the logic behind it.

Reply:
failfastFailureRegexes looks good, will change